client/blocks/plan-storage/bar.jsx

```diff
@@ -58,7 +58,7 @@ class PlanStorageBar extends Component {
 		return (
 			<div className={ classes }>
 				<ProgressBar
-					className="plan-storage__bar"
+					className="plan-storage__storage-bar"
```

Comment:
If we needed that class, I'd rename id to `plan-storage__progress-bar`. However, I don't think it's necessary, so we can totally remove the `className` here.